docs/environment-and-software/user-environment/package-managers/pixi.md

```diff
@@ -0,0 +1,197 @@
+# Using `Pixi` for Package Management
+
+NCAR system users can also use [**`Pixi`**](https://pixi.sh/latest/), a fast package manager built in Rust that uses the conda ecosystem. 
+Pixi provides a modern, project-based workflow for
+managing dependencies across Python, R, C/C++, Julia, and other
+languages, with significantly faster performance than traditional conda.
+
+Pixi builds upon the foundation of the Conda ecosystem, introducing a workspace-centric approach rather than focusing solely on environments.
+This project workspace approach offers a more organized and efficient way to manage dependencies and run code, tailored to modern development practices.
+
+!!! note "uv vs. Pixi vs. Conda"
+    Pixi uses the same package repositories as Conda (conda-forge, etc.)
+    but with much faster dependency resolution and environment creation. If
+    you currently use Conda and find it slow, Pixi may be an excellent
+    alternative. For pure Python projects, consider `uv`
+    instead.
+
+
+## Using `pixi` on NCAR Systems
+
+Pixi is available as an environment module on NCAR systems.
+
+Before using `pixi`, ensure no conda environments are activated by running:
+```bash
+conda deactivate
+```
+
+Then, load the `pixi` module:
+```bash
+module load pixi
+```
+
+The module will automatically unload the system Conda modules to prevent conflicts.
+
+This module:
+
+- provides access to the `pixi` command.
+- configures Pixi to store cache and package data in your
+  `/glade/work` space to avoid filling your home directory quota.
+- allows you to create fast, multi-language environments for your research.
+- supports reproducible workflows through automatic lock file generation.
+
+!!! note "Cache location"
+    Unlike default Pixi behavior, the module redirects all cache locations to your `/glade/work/$USER` directory. This prevents your
+    home directory quota from being exceeded by package caches. You do not
+    need to configure this yourself.
+
+You can verify that Pixi is properly loaded:
+```bash
+    pixi info
+```
+The cache directory should point to your work space, not your home directory.
+
+
+### Creating your own Pixi project
+
+To create a new project:
+
+```bash
+cd /glade/work/$USER/projects
+pixi init example-analysis
+cd example-analysis
+```
+This creates a project directory with a `pixi.toml` configuration file:
```

Comment:
Maybe we should mention that pixi can also create and/or interact with a `pyproject.toml` file as well for environment management?

Reply:
Perfect! Added some more information about `pyproject.toml`.